### ISSUE TITLE
Add new calculator operations in the Calc page

### ISSUE THREAD

**Issue by PaceHut** [PaceHut/hut-calci#1]
New Calculator operations are required in the Calc page, atleast **2** new operations 

And **Do mention** those operations when creating a **_PR_**

**Comment by sambhavgupta0705** [comment]
Hey, I have already made a calculator so my familiar with its code,so can work on it easily

**Comment by PaceHut** [comment]
yes, you can work on this and keep it and do create a PR after 1st Oct on start of Hacktober-Fest

**Comment by sambhavgupta0705** [comment]
Yes will create the PR after 1st October👍

**Comment by yosaami** [comment]
hey, @faaiz-space I want to work on this, please assign me this project.
